src/Service/OnlineCollectionServiceInterface.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace OpenBibIdApi\Service;
+
+interface OnlineCollectionServiceInterface extends ServiceInterface
+{
+
+    /**
+     * Get an online collection by id.
+     *
+     * @param string $collectionId
+     *   The online collection id.
+     *
+     * @return \DOMDocument|null
+     *   A \DOMDocument containing the XML from the response, null if HTTP
+     *   status code 204 (No content) was returned.
+     *
+     * @throws BibApi\Exception\BibException
+     *   When any of the 400, 401, 403, 404 or 421 status codes were returned.
+     */
+    public function getOnlineCollectionById($collectionId);
+
+    /**
+     * Get an online collection by its consumer key.
+     *
+     * @param string $consumerKey
+     *   The online collection consumer key.
+     *
+     * @return \DOMDocument|null
+     *   A \DOMDocument containing the XML from the response, null if HTTP
+     *   status code 204 (No content) was returned.
+     *
+     * @throws BibApi\Exception\BibException
```

Comment:
@throws moet enkel toegevoegd worden in de documentatie als de methode zelf een exception throwed.